tests/pipeline/test_uncensored_prepared.py

```
@@ -0,0 +1,50 @@
+
+    n_patients = len(triggers)
+    n_match = np.sum(triggers == outcomes)
+    n_mismatch = n_patients - n_match
+
+    # Mismatch groups
+    trigger_no_outcome = np.sum((triggers == 1) & (outcomes == 0))
+    outcome_no_trigger = np.sum((triggers == 0) & (outcomes == 1))
+
+    if n_mismatch == 0:
+        print(
+            f"✓ Uncensored scenario: Each patient with a positive outcome has an outcome in the sequence"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)